### PR TITLE
[14.0][MIG] hr_attendance_autoclose

### PR DESCRIPTION
**Pull request by aphon61bank**
Standard migration.

### CODE REVIEW

File: hr_attendance_autoclose/models/hr_attendance.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    def _check_validity(self):
+        """If this is an automatic checkout the constraint is invalid
+        as there may be old attendances not closed
+        """
+        reason = self.env["hr.attendance.reason"].search(
+            [("code", "=", "S-CO")], limit=1
+        )
+        if not reason:
+            return super(HrAttendance, self)._check_validity()
+        if self.filtered(
```
Comment (jarroyomorales) [inline hr_attendance_autoclose/models/hr_attendance.py:63]: Maybe you could write it as `if reason and self.filtered(...` and remove the previous if

**Review by fshah-initos — APPROVED** [review]
Code review Looks good to me

**Comment by fshah-initos** [comment]
@aphon61bank can you please check the comments?

**Comment by fshah-initos** [comment]
Hi @aphon61bank It would be nice you can have a look at above commenst so that PR can be merged soon.

**Comment by fshah-initos** [comment]
Hi @aphon61bank  Can you please have a look at comments so that we can merge it ASAP.

**Comment by fshah-initos** [comment]
@aphon61bank It would be great if you can have look at the above comments.

**Comment by fshah-initos** [comment]
Hello @aphon61bank It would be really nice if you could have a look at above comments.

**Comment by fshah-initos** [comment]
Hi @aphon61bank are you planning to work further with this PR?

**Comment by petrus-v** [comment]
Hi dear @OCA/human-resources-maintainers,

 I think this PR can be close as long #38 has been merged !

regards
Pierre

**Pull request closed without merging**